lib/ios/AirMaps/AIRMapManager.m

```diff
@@ -562,6 +562,32 @@ - (UIView *)view
     }];
 }
 
+RCT_EXPORT_METHOD(getAddressFromCoordinates:(nonnull NSNumber *)reactTag
+                  withLatitude:(double)latitude
+                  withLongitude:(double)longitude
+                  callback:(RCTResponseSenderBlock)callback)
```

Comment:
To treat this as a promise we need to use `RCTPromiseResolveBlock ` and `RCTPromiseRejectBlock ` in place of`RCTResponseSenderBlock` https://reactnative.dev/docs/native-modules-ios#promises